docs/reference/manifest/runtime.md

```diff
@@ -1,22 +1,17 @@
 ---
 title: Runtime in the manifest file
 description: The Runtime element configures your add-in to use a shared JavaScript runtime for its various components, for example, ribbon, task pane, custom functions.
-ms.date: 05/29/2020
+ms.date: 04/07/2021
 localization_priority: Normal
 ---
 
-# Runtime element (preview)
+# Runtime element
```

Comment:
I think I'd modify this to talk about how the Runtime is used for "shared runtime" and also for "event-based activation". These are the two separate things which definitely apply to Runtime and make more sense then "here's how to use it in Excel, and here's how to use it in Outlook". example:

"For the shared runtime, this element enables the ribbon, task pane, and custom functions to use the same runtime..."
"For event based activation, this element enables your add-in to receive events from Outlook such as a new message created event..."

Then the include note about which Office apps support the shared runtime.

Reply:
I added a different include note for the manifest elements

Reply:
I like this!